packages/lib/gpcircuits/scripts/util.ts

```diff
@@ -0,0 +1,17 @@
+import * as fs from "fs/promises";
+import * as path from "path";
+
+/**
+ * Clears a directory.
+ */
+export async function clearDir(directory: string) {
+  for (const file of await fs.readdir(directory)) {
+    await fs.rm(path.join(directory, file));
+  }
+}
+
+/**
+ * Maximum number of parallel promises to avoid
```

Comment:
... for CircomKit calls
There's a specific scenario which motivates this.  I'm sure more parallel promises would be okay if they were doing cheaper things.